src/web/handlers/FlingGestureHandler.ts

```diff
@@ -17,14 +17,23 @@ export default class FlingGestureHandler extends GestureHandler {
 
   private maxDurationMs = DEFAULT_MAX_DURATION_MS;
   private minVelocity = DEFAULT_MIN_VELOCITY;
-  private minDirectionalAlignment = DEFAULT_MIN_DIRECTION_ALIGNMENT;
+  private directionalAlignmentCone = DEFAULT_DIRECTIONAL_ALIGNMENT_CONE;
   private delayTimeout!: number;
 
   private maxNumberOfPointersSimultaneously = 0;
   private keyPointer = NaN;
 
+  private minimalAlignmentCosine = 0;
+
   public init(ref: number, propsRef: React.RefObject<unknown>): void {
     super.init(ref, propsRef);
+
+    const degToRad = (degrees: number) => (degrees * Math.PI) / 180;
+
+    const alignmentRadians = degToRad(this.directionalAlignmentCone);
+    const maxDirectionalDeviation = alignmentRadians / 2;
+
+    this.minimalAlignmentCosine = Math.cos(maxDirectionalDeviation);
```

Comment:
This is constant for every gesture, no? If so, we can move the calculations outside and assign the result to a constant.

Reply:
Please tell me if this current implementation is what you've had in mind.